librz/core/panels.c

```diff
@@ -5375,39 +5356,65 @@ RZ_API bool rz_load_panels_layout(RzCore *core, const char *_name) {
 	__panel_all_clear(panels);
 	panels->n_panels = 0;
 	__set_curnode(core, 0);
-	char *title, *cmd, *x, *y, *w, *h, *p_cfg = panels_config, *tmp_cfg;
-	int i, tmp_count;
-	tmp_cfg = __parse_panels_config(p_cfg, strlen(p_cfg));
-	tmp_count = rz_str_split(tmp_cfg, '\n');
-	for (i = 0; i < tmp_count; i++) {
-		if (RZ_STR_ISEMPTY(tmp_cfg)) {
+
+	RzJson *json = rz_json_parse(panels_config);
+	if (!json || json->type != RZ_JSON_ARRAY) {
+		free(panels_config);
+		return false;
+	}
+	RzJson *child, *baby;
+	const char *title = NULL, *cmd = NULL;
+	int x = 0, y = 0, w = 0, h = 0;
+	// Configuration stored as an array of JSON objects
+	for (child = json->children.first; child; child = child->next) {
+		if (child->type != RZ_JSON_OBJECT) {
 			break;
 		}
-		title = sdb_json_get_str(tmp_cfg, "Title");
-		cmd = sdb_json_get_str(tmp_cfg, "Cmd");
-		(void)rz_str_arg_unescape(cmd);
-		x = sdb_json_get_str(tmp_cfg, "x");
-		y = sdb_json_get_str(tmp_cfg, "y");
-		w = sdb_json_get_str(tmp_cfg, "w");
-		h = sdb_json_get_str(tmp_cfg, "h");
+		for (baby = child->children.first; baby; baby = baby->next) {
+			if (baby->type != RZ_JSON_INTEGER || baby->type != RZ_JSON_STRING) {
+				continue;
+			}
+			// Get window title and executed command
+			if (strcmp(baby->key, "Title") == 0) {
+				title = baby->str_value;
+			} else if (strcmp(baby->key, "Cmd") == 0) {
```

Comment:
If we are breaking compatibility anyway, we might as well call these lowercase `title` and `cmd` I think.